place_types.md

```diff
@@ -43,17 +43,19 @@ countries, but not all countries.
 |[Village](https://datacommons.org/browser/Village)|A small clustered human settlement smaller than a town. |
 {: .doc-table}
 
-### US-specific administrative divisions
+### U.S.-specific administrative divisions
```

Comment:
I'm not sure if tract is considered an administrative division.  Perhaps "U.S.-specific place types"?

Reply:
Hmm, then that begs the question of why we are adding them here. It seems that everything on this page is administrative divisions, so maybe we should just omit them after all?  Probably we need Julia to weigh in here.

Reply:
I think the titles could probably be more general than "administrative division"?  Perhaps use the "geographical division"?

https://screenshot.googleplex.com/A7Zjg9ZY8wz3bbb

e.g, I'm not sure that every City is an administrative division

Reply:
OK, that makes sense. I'll rename everything to "geographical division". Thanks for the suggestion!

Reply:
Alternative, feel free to defer that to Julia, and just add the new types in this PR (given it is not regressing :) )

Reply:
I do +1 changing it, please ignore my last comment :)